src/prefect/cli/orion.py

```diff
@@ -114,39 +126,76 @@ async def start(
 
     base_url = f"http://{host}:{port}"
 
-    async with anyio.create_task_group() as tg:
-        app.console.print(generate_welcome_blurb(base_url, ui_enabled=ui))
-        app.console.print("\n")
-
-        orion_process_id = await tg.start(
-            partial(
-                run_process,
-                command=[
-                    "uvicorn",
-                    "--app-dir",
-                    str(prefect.__module_path__.parent),
-                    "--factory",
-                    "prefect.orion.api.server:create_app",
-                    "--host",
-                    str(host),
-                    "--port",
-                    str(port),
-                ],
-                env=server_env,
-                stream_output=True,
-            )
+    command = [
+        "uvicorn",
+        "--app-dir",
+        str(prefect.__module_path__.parent),
+        "--factory",
+        "prefect.orion.api.server:create_app",
+        "--host",
+        str(host),
+        "--port",
+        str(port),
+        "--access-log",
+    ]
+
+    pid_file = f"{PREFECT_HOME.value()}/orion.pid"
```

Comment:
Need to use the `/` operator for windows support

```suggestion
    pid_file = str(PREFECT_HOME.value() / "orion.pid")
```